modules/ezcontent_node/modules/ezcontent_smart_article/js/ezcontent_smart_article.js

```diff
@@ -2,10 +2,29 @@
  * @file
  */
 
-(function ($) {
+(function ($, Drupal, drupalSettings) {
 
   "use strict";
 
+  /**
+   * Attaches the JS test behavior to to weight div.
+   */
+  Drupal.behaviors.ezcontent_smart_article = {
+    attach: function (context, settings) {
+      $('.tag-field-wrapper .tag-wrapper li', context).click(function(e) {
+        var tagName = $(this).text();
+        var existingTags = $('.tags-link-field .ui-autocomplete-input').val();
+        var finalTags = '';
+        if(existingTags == '') {
+          finalTags = existingTags + tagName;
+        } else {
+          finalTags = existingTags + ', ' + tagName;
+        }
```

Comment:
Rectify if else formatting.
```suggestion
if (existingTags == '') {
  finalTags = existingTags + tagName;
}
else {
  finalTags = existingTags + ', ' + tagName;
}
```